recipes/gala/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "gala" %}
+{% set version = "0.4.0" %}
+{% set sha256 = "53cf601a23d11f945cddd0545cf910039367f6b14759c5868efdf7bc6e9f4564" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py < 35]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy
```

Comment:
Ah right. Can I specify a numpy range, numpy>=1.11? So that it still gets built correctly when 1.12 comes out?